hubblestack/files/hubblestack_nova/win_reg.py

```diff
@@ -67,7 +67,7 @@ def audit(data_list, tags, debug=False, **kwargs):
                     current = _find_option_value_in_reg(reg_dict['hive'], reg_dict['key'], reg_dict['value'])
                     if isinstance(current, dict):
                         tag_data['value_found'] = current
-                        if False in current.values():
```

Comment:
line 70 evalues to True even when current.values() contains an integer 0. This is not expected behaviour.
line 70 should evaluate to True only if current.values() contains a boolean False